src/applications/claims-status/components/claim-files-tab/FilesNeeded.jsx

```diff
@@ -79,8 +79,18 @@ export default function FilesNeeded({ item, previousPage = null }) {
       <span className="alert-description">{getItemDescription()}</span>
       <div className="link-action-container">
         <Link
-          aria-label={`Details for ${item.displayName}`}
-          title={`Details for ${item.displayName}`}
+          aria-label={
+            cstFriendlyEvidenceRequests
+              ? `About this request for ${item.friendlyName ||
```

Comment:
can we just define this once and use it twice?

Reply:
Following on @dysbo with accessibility testing results. The change works perfectly in NVDA (Chrome & Firefox), Narrator on Edge, and MS Voice Access. I was able to see the accessible labels and control them from Links menus and by voice command.